lib/inets/examples/server_root/conf/httpd.conf

```diff
@@ -212,58 +211,52 @@ SSLCertificateKeyFile /var/tmp/server_root/ssl/ssl_server.pem
 # 3 if the client may present a valid certificate but it is not required to
 #   have a valid CA
 
-SSLVerifyClient 0
+{ssl_verify_client, "0"},
```

Comment:
This option does not exist and should not have that value.  ssl option verify should be used and set to verify_none to be eqvivalent!